flink-next-gen-connector-influxdb/src/test/java/org/apache/flink/streaming/connectors/InfluxDBSinkITCase.java

```diff
@@ -92,15 +92,38 @@ void testIncrementPipeline() throws Exception {
         env.execute();
 
         assertThat(
-                COMMIT_QUEUE,
+                queryWrittenData(influxDBConfig),
                 containsInAnyOrder(EXPECTED_COMMITTED_DATA_IN_STREAMING_MODE.toArray()));
     }
 
-    private StreamExecutionEnvironment buildStreamEnv() {
+    private static StreamExecutionEnvironment buildStreamEnv() {
         final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.setRuntimeMode(RuntimeExecutionMode.STREAMING);
         env.setParallelism(1);
         env.enableCheckpointing(100);
         return env;
     }
+
+    private static Queue<String> queryWrittenData(final InfluxDBConfig influxDBConfig) {
```

Comment:
Maybe use a simple List<String> interface here. TODO for your next branch not here anymore.